### ISSUE TITLE
Whitespace text nodes cannot appear as a child of <table>

### ISSUE THREAD

**Issue by brainstorm1313** [remarkablemark/html-react-parser#155]
I expect correct parse:
`parse('<table>\n<tbody><tr><td>sdfq</td></tr></tbody></table>')`

It works, but Chrome console has error:
Warning: validateDOMNesting(...): Whitespace text nodes cannot appear as a child of <table>. Make sure you don't have any extra whitespace between tags on each line of your source code.

https://jsfiddle.net/2f7tpmx5/1/

**Comment by remarkablemark** [comment]
This is similar to https://github.com/aknuds1/html-to-react/issues/79

Long story short, this parser (like [html-to-react](https://github.com/aknuds1/html-to-react)) creates whitespace text nodes given the input HTML.

The HTML to DOM parser that we use is [html-dom-parser](https://github.com/remarkablemark/html-dom-parser).

Fortunately, this is a warning and not an error. However, it might be worth looking into stripping out the whitespace nodes for this library.

**Comment by remarkablemark** [comment]
Please review #162

**Comment by remarkablemark** [comment]
Published [0.13.0](https://github.com/remarkablemark/html-react-parser/releases/tag/v0.13.0):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm install html-react-parser@0.13.0
```

[yarn](https://yarnpkg.com/package/html-react-parser):

```
yarn add html-react-parser@0.13.0
```
